null_safety_examples/misc/lib/effective_dart/usage_bad.dart

```diff
@@ -374,6 +374,18 @@ class Point0 {
 
 //----------------------------------------------------------------------------
 
+// #docregion late-init-list
+class Point1 {
+  late double x, y;
+  Point1.polar(double theta, double radius) {
+    x = cos(theta) * radius;
```

Comment:
Should we look for a case where the assignment cannot happen in the initializer list?

Reply:
I don't think so. "Effective Dart" is already getting pretty big and cumbersome, so I'm hesitant to start introducing counterexamples to show when guidelines *don't* apply. That road seems to lead towards a lot of verbosity.